server/src/models/planterRegistration.model.ts

```diff
@@ -0,0 +1,54 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model({
+  settings: {
+    idInjection: false,
+    postgresql: {schema: 'public', table: 'planter_registrations'},
+  }
+})
+export class PlanterRegistration extends Entity {
+  @property({
+    type: 'number',
+    postgresql: {
+      columnName: 'planter_id',
+      dataType: 'integer',
+    },
+  })
+  planterId: number;
+
+  @property({
+    type: 'date',
+    required: true,
+    dataType: 'timestamp without time zone',
+    postgresql: {
+      columnName: 'created_at',
+    },
+  })
+  createdAt: string;
+
+  @property({
+    type: 'number',
+  })
+  lat?: number;
+
+  @property({
+    type: 'number',
+  })
+  lon?: number;
+
+  // Define well-known properties here
```

Comment:
is this comment relevant?

Reply:
Not really, just part of the template. I'll remove it.